scripts/blender_load_replay.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+r"""
+Script for loading a GFX replay file into Blender. This allows playing back a trajectory from Habitat in Blender.
+
+See the following tutorial for more information on GFX: https://colab.research.google.com/github/facebookresearch/habitat-sim/blob/main/examples/tutorials/colabs/replay_tutorial.ipynb
+
+Example Usage:
+```
+/Applications/Blender.app/Contents/MacOS/Blender --python src_python/habitat_sim/utils/blender_load_replay.py -- --replay data/replays/episode8362.replay.json --root-dir ./
+```
+- Replace `/Applications/Blender.app/Contents/MacOS/Blender` with your path to blender.
+- Also specify `--settings-path settings.json` for dataset specific blender settings (to avoid manually changing the settings in blender for every replay).
+Example `settings` json with all supported fields:
+```
+{
+  "lights": [
+    {
+      "type": "SUN",
+      "name": "my-light",
+      "energy": 3.0,
+      "location": [1.40667, -2.66486, 6.1511]
+    }
+  ]
+}
+```
+
+"""
+
+
+try:
+    import bpy
+except ImportError:
+    raise ImportError(
+        "Failed to import Blender modules. This script can't run "
+        "standalone. Run `blender --python path/to/blender_load_replay.py ...`. Watch the terminal for "
+        "debug/error output."
+    )
+
+import argparse
+import json
+import os
+from dataclasses import dataclass
+
+
+@dataclass
+class ImportItem:
+    filepath: str
+    do_join_all: bool = True
+    force_color: list = None
+
+
+def import_scene_helper(raw_filepath):
+    """
+    Import a single asset into Blender
+    """
+
+    # These fixups will be tried one at a time, in order, until a matching, existing file is found
+    model_filepath_fixups = [
+        # use uncompressed versions when available
+        ("/stages/", "/stages_uncompressed/"),
+        ("/urdf/", "/urdf_uncompressed/"),
+        None,
+    ]
+
+    filepath = None
+    for fixup in model_filepath_fixups:
+        raw_filepath = raw_filepath.split("?")[0]
+        if fixup is None:
+            filepath = raw_filepath
+        else:
+            if raw_filepath.find(fixup[0]) != -1:
+                filepath = raw_filepath.replace(fixup[0], fixup[1])
+            else:
+                continue
+        if os.path.exists(filepath):
+            break
+    else:
+        raise RuntimeError("can't find file " + raw_filepath, filepath)
+
+    ext = os.path.splitext(filepath)[-1].lower()
+    if ext == ".dae":
+        # Convert to GLB.
+        orig_filepath = filepath
+        filepath = filepath.replace(".dae", ".glb")
+        raw_filepath = raw_filepath.replace(".dae", ".glb")
+        if not os.path.exists(filepath):
+            os.system(f"assimp export {orig_filepath} {filepath}")
```

Comment:
Let's make this more robust. Check if `assimp` is available. If not, print a warning message and instructions for installing Assimp (it's just `pip install assimp`, right?).